app/build.gradle

```diff
@@ -33,11 +33,12 @@ android {
 }
 
 dependencies {
-
     implementation 'androidx.appcompat:appcompat:1.2.0'
     implementation 'com.google.android.material:material:1.3.0'
     implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
     testImplementation 'junit:junit:4.+'
     androidTestImplementation 'androidx.test.ext:junit:1.1.2'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
+    testImplementation 'org.junit.jupiter:junit-jupiter:5.7.2'
```

Comment:
Remove this. We are already using `testImplementation 'junit:junit:4.+'`

Reply:
fixed